src/data/uiConfig.ts

```diff
@@ -141,14 +163,17 @@ export const intervalGainPctTooltipContent = (
     config.cellarNameKey === CellarNameKey.STEADY_BTC ||
     config.cellarNameKey === CellarNameKey.STEADY_ETH ||
     config.cellarNameKey === CellarNameKey.STEADY_UNI ||
-    config.cellarNameKey === CellarNameKey.STEADY_MATIC
-  )
-    return `% change of token price compared to a benchmark portfolio of USDC`
-  if (
+    config.cellarNameKey === CellarNameKey.STEADY_MATIC ||
+    config.cellarNameKey === CellarNameKey.REAL_YIELD_USD ||
     config.cellarNameKey === CellarNameKey.ETH_BTC_MOM ||
     config.cellarNameKey === CellarNameKey.ETH_BTC_TREND
   )
-    return `% change of token price compared to a benchmark portfolio of 50% ETH and 50% BTC`
```

Comment:
are the usdc eth btc chart fixed yet? bring back ETH BTC MOM and TREND to 1M 50/50